DataFormats/L1TGlobal/interface/GlobalObject.h

```diff
@@ -24,6 +24,7 @@ enum GlobalObject
     gtHTT,
     gtHTM,
     gtETMHF,
+    gtTowerCount,
```

Comment:
nothing appears to depend on how many enums of this type there are - so seems ok to me.